lua/telekasten.lua

```diff
@@ -581,6 +582,7 @@ end
 local function PreviewImg(_)
 	vim.cmd("normal yi)")
 	local fname = vim.fn.getreg('"0')
+	fname = fname:gsub("^(%[)(.+)(%])$", "%2")
```

Comment:
This is a `yi)` here for `![](path/to/img.png)`-like links. So there's no need for stripping `[]`s.

Reply:
I will remove that post-merge